src/applications/login/components/SignInButtons.jsx

```diff
@@ -11,67 +11,92 @@ function loginHandler(loginType) {
+        >
+          <img
+            alt="Sign in with Login.gov"
+            src={`${vaGovFullDomain}/img/signin/logingov-icon-white.svg`}
+          />
+        </button>
+      )}
       <button
         disabled={isDisabled}
-        className="dslogon"
-        onClick={() => loginHandler('dslogon')}
+        type="link"
+        aria-label="Sign in with ID.me"
+        className="usa-button usa-button-big idme-button vads-u-margin-y--1p5"
+        onClick={() => loginHandler('idme')}
       >
         <img
-          aria-hidden="true"
-          role="presentation"
-          alt="DS Logon"
-          src={`${vaGovFullDomain}/img/signin/dslogon-icon.svg`}
+          alt="Sign in with ID.me"
+          src={`${vaGovFullDomain}/img/signin/idme-icon-white.svg`}
         />
-        <strong> Sign in with DS Logon</strong>
       </button>
       <button
         disabled={isDisabled}
-        className="mhv"
-        onClick={() => loginHandler('mhv')}
+        type="link"
+        aria-label="Sign in with DS Logon"
+        className="usa-button usa-button-big dslogon-button vads-u-margin-y--1p5"
+        onClick={() => loginHandler('dslogon')}
       >
-        <img
-          aria-hidden="true"
-          role="presentation"
-          alt="My HealtheVet"
-          src={`${vaGovFullDomain}/img/signin/mhv-icon.svg`}
-        />
-        <strong> Sign in with My HealtheVet</strong>
+        DS Logon
       </button>
       <button
         disabled={isDisabled}
-        className="usa-button-primary va-button-primary"
-        onClick={() => loginHandler('idme')}
+        type="link"
+        aria-label="Sign in with My HealtheVet"
+        className="usa-button usa-button-big mhv-button vads-u-margin-y--1p5"
+        onClick={() => loginHandler('mhv')}
       >
-        <img
-          aria-hidden="true"
-          role="presentation"
-          alt="ID.me"
-          src={`${vaGovFullDomain}/img/signin/idme-icon-white.svg`}
-        />
-        <strong> Sign in with ID.me</strong>
+        My HealtheVet
       </button>
-      <span className="sidelines">OR</span>
       <div className="alternate-signin">
-        <h2 className="vads-u-font-size--sm vads-u-margin-top--0">
-          Don't have those accounts?
-        </h2>
-        <button
-          disabled={isDisabled}
-          className="idme-create usa-button usa-button-secondary"
-          onClick={() => signup('v1')}
-        >
-          <img
-            aria-hidden="true"
-            role="presentation"
-            alt="ID.me"
-            src={`${vaGovFullDomain}/img/signin/idme-icon-dark.svg`}
-          />
-          <strong> Create an ID.me account</strong>
-        </button>
-        <p>Use your email, Google, or Facebook</p>
+        <h2 className="vads-u-margin-top--3">Or create an account</h2>
+        <div className="vads-u-display--flex vads-u-flex-direction--column">
+          {loginGovEnabled && (
+            <a
+              role="link"
```

Comment:
`role="link"` isn't needed here because it's already an anchor element, however it does need a valid `href`. Otherwise, let's make it a button

Reply:
@timwright12 So would you like me to overwrite all the CSS for the button so that it looks like the Action link?

Reply:
Lemme pull @bkjohnson and @k80bowman for some guidance. I'm sure there's something we can do in the confines of the system that don't violate WCAG

Reply:
@k80bowman Yes.  Using the Action links were a part of the feedback my team received from the Collab cycle and direction from the DEPO design lead.  And I'm willing to take back this information to my DEPO leads as the intention is to navigate to a page to create an account.

Reply:
@AGarcia-Clarity Oh interesting. Yeah, if you wouldn't mind taking this back to them, I'd appreciate it. And please let them know they can reach out to us on the Design System team with any questions.

Reply:
action links are not buttons. they should be treated and built as links.